c10/test/util/MPMCQueue_test.cpp

```diff
@@ -0,0 +1,73 @@
+#include <thread>
```

Comment:
I guess new files here are automatically picked up?

Reply:
Yeah I took another look and seems we have a glob rule https://github.com/pytorch/pytorch/blob/main/c10/test/CMakeLists.txt